app/src/main/java/org/mozilla/fenix/search/SearchFragment.kt

```diff
@@ -43,6 +43,11 @@ class SearchFragment : Fragment() {
         return view
     }
 
+    override fun onResume() {
+        super.onResume()
+        toolbarComponent.editMode()
```

Comment:
Once you rebase master we shouldn't need this anymore since we're starting the component in edit mode

https://github.com/mozilla-mobile/fenix/blob/0a4bc1c55a68227a79d720a265a81ee3dc50ff5b/app/src/main/java/org/mozilla/fenix/search/SearchFragment.kt#L39